### ISSUE TITLE
LiveLambda: Large WebSocket response throw IllegalLocationConstraintException in new regions

### ISSUE THREAD

**Issue by TerenzChan** [serverless-stack/serverless-stack#1399]
S3 is used to process large WebSocket responses (e.g. in `ws.ts`), but the region option is not explicitly set `const s3 = new S3();`.
Therefore IllegalLocationConstraintException was thrown, as that is required for those new regions.

Please add an option to allow setting the region option, or pass the region of the stack to the handler.

**Comment by fwang** [comment]
Hi @TerenzChan, I'm taking a look at this.

> const s3 = new S3();
Are you referencing the stub function here https://github.com/serverless-stack/serverless-stack/blob/master/packages/resources/assets/stub/index.js#L17?

Btw which region did you try with?

**Comment by TerenzChan** [comment]
> Hi @TerenzChan, I'm taking a look at this.
> 
> > const s3 = new S3();
> > Are you referencing the stub function here https://github.com/serverless-stack/serverless-stack/blob/master/packages/resources/assets/stub/index.js#L17?
> 
Seems like that will exist in those files that used S3.
I found it in the stub function you posted and 
[https://github.com/serverless-stack/serverless-stack/blob/master/packages/core/src/runtime/ws.ts#L7](url)

> Btw which region did you try with?

I used ap-east-1.

It looks like the global endpoint of S3 will be used if the region option is not set. And the request will not be redirected to the correct region automatically for those buckets in opt-in regions, which were launched after March 20, 2019.
